packages/dendron-next-server/pages/workspace/config.tsx

```diff
@@ -0,0 +1,326 @@
+        siteNotesDir: "",
+        siteRepoDir: "",
+        siteRootDir: "docs",
+        usePrettyRefs: false,
+      },
+    },
+  };
+};
+
+// const saveConfigData = (data: DendronConfig) => {
+//   console.log("data saved");
+//   console.log(JSON.stringify(data, null, 2));
+// };
+
+type InputControlProps = {
+  name: string;
+  label?: string;
+  placeholder: string;
+  help: string | ReactNode;
+  disabled?: boolean;
+};
+
+function InputControl({
+  name,
+  label,
+  placeholder,
+  help,
+  disabled,
+}: InputControlProps) {
+  return (
+    <Field name={name}>
+      {({ form: { errors, touched } }) => (
+        <FormControl
+          isInvalid={
+            /**
+             * If field has errors AND they've interacted with this field, mark
+             * control as invalid. */
+            !!(get(errors, name) && get(touched, name))
+          }
+        >
+          {!!label && <FormLabel htmlFor={name}>{label}</FormLabel>}
+
+          <Field
+            //
+            as={Input}
+            name={name}
+            placeholder={placeholder}
+            id={name}
+            disabled={disabled}
+          />
+
+          <FormErrorMessage>{errors.site?.siteRootDir}</FormErrorMessage>
+
+          <FormHelperText>{help}</FormHelperText>
+        </FormControl>
+      )}
+    </Field>
+  );
+}
+
+export default function ConfigSamplePage() {
+  // const { data: configData } = getConfigData();
+
+  const [configData, saveConfigData] = useState(getConfigData().data);
+
+  React.useEffect(() => {
+    console.log("save", JSON.stringify(configData, null, 2));
+  }, [JSON.stringify(configData)]);
+
+  return (
+    <Container paddingBottom={12}>
+      <Stack spacing={4}>
+        <Heading>Config Data</Heading>
+
+        <Formik
+          enableReinitialize
+          initialValues={configData}
+          onSubmit={(formValues) => saveConfigData(formValues)}
+        >
+          {({
+            touched,
+            errors,
+            values,
+            resetForm, // NOTE only works on fields which have configData keys
+          }) => (
+            <Box as={Form} onChange={console.log}>
+              <Stack spacing={8}>
+                <Box as="fieldset">
+                  <Heading size="md" as="legend">
+                    Vaults
+                  </Heading>
+
+                  <Stack spacing={4}>
+                    {configData.vaults.map((vault, index) => (
+                      <Tooltip
+                        label="Vaults are not yet editable via this form."
+                        placement="top"
+                        hasArrow
+                      >
+                        <Stack direction="row" spacing={2}>
+                          <FormLabel
+                            htmlFor={`vaults[${index}].fsPath`}
+                            whiteSpace="nowrap"
+                            paddingTop={2} // align with text field
+                          >
+                            Vault {index}
```

Comment:
Since the index of the vaults don't carry any significance (yet), can we drop this label or make it empty?

Reply:
:heavy_check_mark: @ 2177b0f2